### PR TITLE
Add Berlin2

### PR DESCRIPTION
**Pull request by RosemaryOrchard**
Added Berlin2 (copied from Berlin, dates and links changed)

### CODE REVIEW

File: templates/berlin2/schedule.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<p>
+  <i class="calendar icon"></i> <time class="dt-start" datetime="2019-05-04">May 4</time>-<time class="dt-end" datetime="2019-05-05">5</time>, 2019<br>
```
Comment (tantek) [inline templates/berlin2/schedule.php]:
```suggestion
  <i class="calendar icon"></i> <time class="dt-start" datetime="2019-11-23">November 23</time>-<time class="dt-end" datetime="2019-11-24">24</time>, 2019<br>
```

File: templates/berlin2/schedule.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      <span class="featured">Learn, Share, Discuss!</span>
+    </div>
+    <div class="summary">
+      Day 1 is about discussing in a BarCamp-like environment. Bring a topic you'd like to discuss or join in on topics as they are added to the board.
+    </div>
+    <div class="details">
+      <ul>
+       <li><span class="time">9:00</span> Organizer setup</li>
+       <li><span class="time">9:30</span> Doors Open</li>
+       <li><span class="time">10:30</span> Introduction</li>
```
Comment (tantek) [inline templates/berlin2/schedule.php]:
```suggestion
       <li><span class="time">10:00</span> Introduction</li>
```

File: templates/berlin2/schedule.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+       <li><span class="time">10:45</span> Keynote</li>
```
Comment (tantek) [inline templates/berlin2/schedule.php]:
```suggestion
       <li><span class="time">10:15</span> Keynote</li>
```

File: templates/berlin2/schedule.php [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      Sunday, November 24<br>
+      <span class="featured">Create, Hack, Demo!</span>
+    </div>
+    <div class="summary">
+      Day 2 is about implementing what we discussed. Work with others or on your own. We'll have demos at the end of the day to see what everyone worked on!
+    </div>
+    <div class="details">
+      <ul>
+       <li><span class="time">9:00</span> Organizer setup</li>
+       <li><span class="time">10:00</span> Day 2 kick-off</li>
```
Comment (tantek) [inline templates/berlin2/schedule.php:62]:
```suggestion
       <li><span class="time">9:30</span> Doors Open</li>
       <li><span class="time">10:00</span> Day 2 kick-off</li>
```

File: templates/berlin2/schedule.php [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+       <li><span class="time">17:45</span> Clean-up</li>
+       <li><span class="time">18:00</span> Camp closed</li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+<br>
+
+<p>Schedule is subject to change. View the <a href="https://indieweb.org/2019/Berlin/Schedule">full schedule grid</a> on the IndieWeb wiki for the latest updates.</p>
```
Comment (tantek) [inline templates/berlin2/schedule.php]:
```suggestion
<p>Schedule is subject to change. View the <a href="https://indieweb.org/2019/Berlin2/Schedule">full schedule grid</a> on the IndieWeb wiki for the latest updates.</p>
```

**Review by tantek — CHANGES_REQUESTED** [review]
Suggested a bunch of fixes. Take a look.

**Review by RosemaryOrchard — COMMENTED** [review]
Commit all changes I hope?

**Review by tantek — APPROVED** [review]
Looks good!

**Review by RosemaryOrchard — COMMENTED** [review]
Fix duplicate doors opening

**Pull request merged**